pkg/virt-handler/virtwrap/schema.go

```diff
@@ -85,6 +108,7 @@ type Domain struct {
 
 type DomainStatus struct {
 	Status LifeCycle
+	Reason LifeCycle
```

Comment:
Yes, please duplicate them, making such things explicit will save us trouble. I still don't like that we can freely mix reasons and states and that there is no validation, but since only the watch loop should ever do this mapping and no other place in the project, it should be ok for now ...